src/lib/snyk-test/iac-test-result.ts

```diff
@@ -1,7 +1,9 @@
+import pick = require('lodash.pick');
```

Comment:
I'd suggest using the standard import syntax:

```suggestion
import * as pick from 'lodash.pick';
```